content/01-responsive-web-design/basic-css/attach-a-fallback-value-to-a-css-variable.md

```diff
@@ -37,7 +40,7 @@ assert(
 );
 ```
 
-The fallback value of `black` should be used in `background` property of the `penguin-bottom` class.
+O valor substituto de `black` deve ser usado na propriedade seguir da classe `penguin-bottom`. 
```

Comment:
Please, review this sentence... There's no make sense... At least for me...

Reply:
fallback é um pouco dificil de se traduzir , eu pensei em colocar valor substituto ou valor retornado não sei qual seria melhor

Reply:
It isn't about fallback. Sorry?... 

It's about the `background` property. I think you forgot to put the property name in this sentence.

Your translation:

> [...] deve ser usado na propriedade seguir da classe `penguin-bottom`.

My suggestion to replace:

> [...] deve ser usado na propriedade `background` da classe `penguin-bottom`.

Reply:
justo